client/src/components/RuleBuilder/SavedRulesSelector.vue

```diff
@@ -42,11 +42,20 @@ export default {
         numOfSavedRules: function () {
             return this.savedRules.length;
         },
+        sortSavedRules: function () {
+            var sortedRules = this.savedRules;
+            return sortedRules.sort(this.onSessionDateTime);
```

Comment:
That's still sorting the prop in place, isn't it (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) ?

```suggestion
            return [...this.savedRules].sort(this.onSessionDateTime);
```
is my totally untested suggestion

Reply:
I tested it! 👍🏻